lib/spack/spack/build_systems/rocm.py

```diff
@@ -71,79 +71,23 @@
 #
 # 3. In part 2, DO NOT list the path to hsa as /opt/rocm/hsa ! You want spack
 #    to find hsa in /opt/rocm/include/hsa/hsa.h . The directory of
-#    /opt/rocm/hsa also has an hsa.h file, but it won't be found because spack
+#    /opt/rocm/hsa also has an hsa.h file, but it won"t be found because spack
 #    does not like its directory structure.
 #
 
-import spack.variant
-from spack.directives import conflicts, depends_on, variant
+from spack.directives import *
 from spack.package_base import PackageBase
 
 
 class ROCmPackage(PackageBase):
-    """Auxiliary class which contains ROCm variant, dependencies and conflicts
-    and is meant to unify and facilitate its usage. Closely mimics CudaPackage.
+    """Auxiliary class which contains ROCm variant, and ROCm meta tools
+    and is meant to unify and facilitate its usage.
```

Comment:
The phrasing of this sentence with its multiple uses of `and` make it hard to read. I suggest a more verbose but more descriptive wording. Perhaps the following.

```suggestion
    """This specialized package base class provides a ROCm variant
    and meta tools to package classes that derive from it. It is meant
    to unify and facilitate the specification of ROCm compilation option
    to packages that can use it.
```